src/eastwood/lint.clj

```diff
@@ -460,10 +460,13 @@ Return value:
                    :rethrow-exceptions? false})
 
 (defn last-options-map-adjustments [opts reporter]
-  (let [opts (merge default-opts opts)
+  (let [{:keys [namespaces] :as opts} (merge default-opts opts)
+        distinct* (fn [x] ;; distinct but keeps original coll type
+                    (->> (into (empty x) (distinct) x)
```

Comment:
I might have preferred a non-transducer version, as the transducer one makes me think you've misplaced a paren.

Reply:
I agree, Chose the transducer works with sets;

```clj
user=> (let [x #{1 2 3}] (into (empty x) (distinct x)))
Execution error (UnsupportedOperationException) at user/eval136 (REPL:2).
nth not supported on this type: PersistentHashSet
```